transitions/conditional_alerts.js

```diff
@@ -48,20 +48,12 @@ module.exports = {
             });
         }
     },
-    _hasRun: function(doc) {
-        return Boolean(
-            doc &&
-            doc.transitions &&
-            doc.transitions.conditional_alerts
-        );
-    },
     filter: function(doc) {
         var self = module.exports;
         return Boolean(
             doc &&
             doc.form &&
             doc.type === 'data_record' &&
-            self._hasConfig(doc) &&
```

Comment:
Nice catch - without this any unconfigured reports will rerun every time they change!